### DISCUSSION TITLE
Tutanota Email

### DISCUSSION THREAD

**Discussion by iamk3** [moralmunky/Home-Assistant-Mail-And-Packages#532] [Q&A] [answered]
Has anyone had experience using Tutanota for this integration? The sign in did not work on the port and I'm not sure where to go from here. Any help will be appreciated!

Thanks!

**Comment by firstof9** [comment, marked as answer]
Doesn't look like they support IMAP so this won't work.

Refrence: https://tutanota.com/faq#imap

**Reply by iamk3** [reply]
Thank you. I didn't realize it was IMAP. Any suggestions on a good private provider?

**Reply by firstof9** [reply]
I don't have any, I use gmail.

**Reply by iamk3** [reply]
Okay. Thanks!
